lms/templates/verify_student/test/fake_softwaresecure_response.html

```diff
@@ -56,9 +58,9 @@
 
         $.ajax({
             type: "POST",
-            url: '${results_callback}',
+            url: '${results_callback | n, js_escaped_string}',
             headers: {
-            "Authorization": "${authorization_code}"
+            "Authorization": "${authorization_code | n, js_escaped_string}"
             },
             data: JSON.stringify(data),
```

Comment:
Sorry, I didn't notice it was a test file. Definitely don't need to worry about it.